website/siteConfig.js

```diff
@@ -66,7 +66,13 @@ const siteConfig = {
   scripts: [
     "https://cdn.jsdelivr.net/npm/focus-visible@5.0.2/dist/focus-visible.min.js",
     "https://platform.twitter.com/widgets.js",
-    "https://buttons.github.io/buttons.js"
+    "https://buttons.github.io/buttons.js",
+    'https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.0/clipboard.min.js',
+    '/react-native-windows/js/code-block-buttons.js',
+  ],
+
+  stylesheets: [
```

Comment:
Does this not get included automatically just be adding it into the static css folder?

Reply:
apparently not 🙃 the docusaurus docs point to that gist and the gist said to include it. @slorber do you know?

Reply:
I don't really know 😅 started working on Docusaurus mostly on v2 and all this is quite old. I guess the gist it right